.github/workflows/build.yml

```diff
@@ -239,6 +239,52 @@ jobs:
       - name: Meson - Uninstall
         run: ninja -C build uninstall
 
+  build-archlinux:
+    name: Archlinux
+    runs-on: ubuntu-22.04
+    container:
+      image: archlinux:latest
```

Comment:
Have you tried using an earlier "stable" image? I wonder if they broke something with tracker in "latest"?